tools-v2/pkg/cli/command/curvebs/list/maydamagedvolumes/maydamagedvolumes.go

```diff
@@ -0,0 +1,162 @@
+package maydamagedvolumes
+
+import (
+	"fmt"
+	"time"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/proto/common"
+	"github.com/opencurve/curve/tools-v2/proto/proto/topology"
+	"github.com/spf13/cobra"
+)
+
+const (
+	Exam = "curve bs list may-damaged-vols"
+)
+
+type MayDamVolCmd struct {
+	basecmd.FinalCurveCmd
+	timeout    time.Duration
+	retryTimes int32
+	addrs      []string
+
+	clusterPoolsetsInfo []*topology.PoolsetInfo
+	listPoolsetsRpc     *ListPoolsetsRpc
+
+	clusterPhyPoolsInfo  []*topology.PhysicalPoolInfo
+	listPhyPoolsInPstRpc *ListPhyPoolsInPstRpc
+
+	clusterZonesInfo []*topology.ZoneInfo
+	listZonesRpc     *ListZonesRpc
+
+	clusterServersInfo []*topology.ServerInfo
+	listServersRpc     *ListServersRpc
+
+	listChunkServersRpc *ListChunkServersRpc
+	chunkServerInfos    []*topology.ChunkServerInfo
+	offLineCsAddrs      []string
+
+	getChunkInfoRpc             *GetChunkInfoRpc
+	getCopySetsInChunkServerRpc *GetCopySetsInChunkServerRpc
+
+	copysetInfos []*common.CopysetInfo
+	copysetsCnt  map[uint32]int
+	damagedCps   []*common.CopysetInfo
+
+	listVolsOnCpysRpc *ListVolumesOnCopysetsRpc
+	fileNames         []string
+	isRetired         bool
+
+	rows []map[string]string
+}
+
+func NewMayDamVolCmd() *cobra.Command {
+	mdvCmd := &MayDamVolCmd{
+		FinalCurveCmd: basecmd.FinalCurveCmd{
+			Use:     "may-damaged-vols",
```

Comment:
damaged-vols may be better？

Reply:
> `damaged-vols` may be better？

no, original name is may-broken-vol, may can't be removed